packages/components/pie-radio-group/test/visual/pie-radio-group.spec.ts

```diff
@@ -1,11 +1,77 @@
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
+import {
+    type PropObject, type WebComponentPropValues, type WebComponentTestInput,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/defs.ts';
+import {
+    getAllPropCombinations, splitCombinationsByPropertyValue,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/get-all-prop-combos.ts';
+import { createTestWebComponent } from '@justeattakeaway/pie-webc-testing/src/helpers/rendering.ts';
+import { WebComponentTestWrapper } from '@justeattakeaway/pie-webc-testing/src/helpers/components/web-component-test-wrapper/WebComponentTestWrapper.ts';
+import { percyWidths } from '@justeattakeaway/pie-webc-testing/src/percy/breakpoints.ts';
+import { PieRadio } from '@justeattakeaway/pie-radio';
+import { PieFormLabel } from '@justeattakeaway/pie-form-label';
 import { PieRadioGroup } from '../../src/index.ts';
 
-test.describe('PieRadioGroup - Visual tests`', () => {
-    test('should display the PieRadioGroup component successfully', async ({ page, mount }) => {
-        await mount(PieRadioGroup);
+const props: PropObject = {
```

Comment:
I think you can use the generic typing here `PropObject<RadioGroupProps>`

Reply:
`hasGroupLabel` isn't a valid prop for the component itself but needed for the tests so we will get a type error. i would keep it automatically inferred

Reply:
updated